test/e2e/network/conntrack.go

```diff
@@ -117,7 +132,7 @@ var _ = SIGDescribe("Conntrack", func() {
 		ginkgo.By("creating a client pod for probing the service " + serviceName)
 		clientPod := newAgnhostPod(podClient, "")
 		clientPod.Spec.NodeName = clientNodeInfo.name
-		cmd := fmt.Sprintf(`date; for i in $(seq 1 300); do echo "$(date) Try: ${i}"; echo hostname | nc -u -w 5 -p %d %s %d; echo; done`, srcPort, serverNodeInfo.nodeIP, udpService.Spec.Ports[0].NodePort)
+		cmd := fmt.Sprintf(`date; for i in $(seq 1 3000); do echo "$(date) Try: ${i}"; echo hostname | nc -u -w 5 -p %d %s %d; echo; done`, srcPort, serverNodeInfo.nodeIP, udpService.Spec.Ports[0].NodePort)
```

Comment:
unrelated to your PR but when I added this command I was thinking that each iteration was 5 seconds (`nc -w` says it waits), however, the logs show multiple executions in the same second
```Fri Jun 19 11:18:54 UTC 2020 Try: 5

Fri Jun 19 11:18:54 UTC 2020 Try: 6

Fri Jun 19 11:18:54 UTC 2020 Try: 7
```
this is much better for sure